web/controllers/vote_controller.ex

```diff
@@ -0,0 +1,47 @@
+defmodule SignDict.VoteController do
+  use SignDict.Web, :controller
+
+  plug Guardian.Plug.EnsureAuthenticated, handler: SignDict.GuardianErrorHandler
+
+  alias SignDict.Vote
+  alias SignDict.Video
+
+  def create(conn, %{"video_id" => video_id}) do
+    IO.puts "creating"
```

Comment:
@EinLama thanks for reviewing. You were right, this was debug output. 
(This)[https://github.com/signdict/website/pull/56/commits/5b92235f7e933419dfaaff19b45647620d9f6d9d] commit solves it 🎉